stations.csv

```diff
@@ -8563,7 +8562,6 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 10436;Les Bons-Enfants;les-bons-enfants;8775116;;;;;FR;Europe/Paris;f;f;f;f;f;f;FRHXA;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv10436;
 10437;Montfort;montfort;8775117;;44.0666670000;5.9666670000;;FR;Europe/Paris;f;f;f;f;f;f;FRHXB;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv10437;
 10438;L’Escale;lescale;8775118;;;;;FR;Europe/Paris;f;f;f;f;f;f;FRHXC;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv10438;
-10439;Rodange Frontière;rodange-frontiere;8719494;;49.551000;5.843539;8606;LU;Europe/Luxembourg;f;f;f;f;f;f;FRBTS;;t;;f;8271142;f;;f;;f;;f;;f;;f;8271142;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv10439;
```

Comment:
@lucavezz will be unhappy because of the Trenitalia ID being removed :sweat_drops:

Reply:
Ah! @lucavezz sorry. I will add it to another Rodange station, would that be better @misdoro ?

Reply:
I only see OBB and DB ids for this stations. I'll check with @lucavezz

Reply:
Ah, I miscalculated the columns then, sorry.
I'm wondering why the station was not enabled for DB in that case

Reply:
Hum, searching Rodange to Koln seem to work fine, yielding DB results

Reply:
The other Rodange stations do have db and obb ids and they are enabled. My worry is that they are not the same ids as the ones I removed. But then again, if they were not enabled, we never actually use those ids.